algosdk/atomic_transaction_composer.py

```diff
@@ -499,21 +499,22 @@ def execute(
 
             if i not in self.method_dict:
                 continue
-            # Return is void
-            if self.method_dict[i].returns.type == abi.Returns.VOID:
-                method_results.append(
-                    ABIResult(
-                        tx_id=tx_id,
-                        raw_value=raw_value,
-                        return_value=return_value,
-                        decode_error=decode_error,
-                        tx_info=client.pending_transaction_info(tx_id),
-                    )
-                )
-                continue
 
             # Parse log for ABI method return value
             try:
+                # Return is void  but, we still want to get the tx_info for this transaction
+                if self.method_dict[i].returns.type == abi.Returns.VOID:
+                    method_results.append(
+                        ABIResult(
+                            tx_id=tx_id,
+                            raw_value=raw_value,
+                            return_value=return_value,
+                            decode_error=decode_error,
+                            tx_info=client.pending_transaction_info(tx_id),
```

Comment:
nit: might as well put `resp = client.pending_transaction_info(tx_id)` at the top of the try block and use `resp` here instead of having two calls to `client.pending_transaction_info`

Reply:
👍